zipline/__main__.py

```diff
@@ -51,6 +59,17 @@ def main(extension, strict_extensions, default_extension):
         os.environ,
     )
 
+    for c in custom_types:
```

Comment:
Right now the user still needs to add an argument to `run` to handle any of these. These are also known at import time, so could we move this code to module scope or make it less automatic?